.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - lts/dubnium
+  - lts/carbon
```

Comment:
Could we keep `lts/dubnium` and add `lts/carbon` to the list?

Reply:
I’m personally fine with only testing the outliers. `node/lts` seems to resolve to carbon currently, that’s a good alternative?

Reply:
Personally, I like testing across latest and all of the active LTS releases just in case.
But I'd be fine with testing just latest and any known outliers.

Reply:
`node/lts` would allow it to drift outside of your control, if that's what you want.

Reply:
I like drifting, because it’s less commits to maintain, but explicit is also fine, I don’t think I have ever benefitted from Travis finding a bug between the outliers. Anyone, no strong feelings 🤷‍♂️